lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -300,54 +318,93 @@ llvm::Error Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
         {idim.n, outSz.first, outSz.second, depth}};
 
     TypeRef outTy;
-    Constant *filter = W;
-    Constant *bias;
+    Storage *filter = nullptr, *bias = nullptr;
     if (typeName == "Conv" || typeName == "ConvRelu") {
-      // Construct the Bias field.
-      Constant *biasConstant = nullptr;
-
-      // Check if we have a serialized bias.
-      if (op.input_size() > 2) {
-        const auto &biasName = op.input(2);
-        biasConstant = getConstantByNameOrNull(biasName);
-      }
-
-      // If no serialized bias was found then create a zero bias.
-      if (!biasConstant) {
+      if (bindings) {
+        filter = createPlaceHolderFromWeightTensor(
+            bindings, G_, &W->getPayload(), "conv.filter", false /*move*/,
+            false /*quantized*/, [&kernels, &idim, this](Tensor *t) {
```

Comment:
Use the exact same argument names in the inline comments as you did in the function declaration/definition (e.g. `isQuantized`).